assets/src/scss/core-overrides/embed.scss

```diff
@@ -0,0 +1,132 @@
+// This combination of height: 0, and padding-bottom: 56.25%
+// is to force the aspect ratio of the video to 16:9, based
+// on the width. See: https://css-tricks.com/fluid-width-video/
+@mixin embed-wrapper {
+  position: relative;
+  overflow: hidden;
+  max-width: 100%;
+  max-height: 100%;
+  padding-bottom: 56.25%;
+  height: 0;
+}
+
+@mixin absolute-position {
```

Comment:
We might need this mixin for other blocks 🤔 Maybe we could move it to the `_mixins.scss` file?

Reply:
I actually see we have this [fill-container](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_mixins.scss#L93-L100) mixin in that file, doesn't it do the same thing?

Reply:
Okay, thanks for testing! Do you think it makes sense to move this one to the mixins file or not?